packages/react-core/src/components/DataList/DataListItemRow.tsx

```diff
@@ -1,20 +1,23 @@
 import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/DataList/data-list';
+import { WrapModifier } from './DataList';
 
 export interface DataListItemRowProps extends Omit<React.HTMLProps<HTMLDivElement>, 'children'> {
```

Comment:
Looks like the prop descriptions were missing here.  Do you mind adding those as part of your PR?